.github/workflows/prod.yml

```diff
@@ -6,7 +6,9 @@ on:
     types:
       - released
       - prereleased
-
+  push:
+    branches:
+      - ft-chang-service-port
```

Comment:
@Mubangizi why are we doing a direct production push/deploy from this branch. Any reason for not following the conventional workflow?

Reply:
Wow.